app/services/widget_import_service.rb

```diff
@@ -25,6 +25,12 @@ def import_widget_from_hash(widget)
     log_widget_import_message(new_or_existing_widget)
 
     new_or_existing_widget.update_attributes(widget)
+    if new_or_existing_widget.miq_schedule
```

Comment:
This looks good but, maybe it would be better to move this logic into the `build_miq_schedule` method [here](https://github.com/ManageIQ/manageiq-ui-classic/blob/7d62772729b9212cda4a9171cded5f1af59e1dd7/app/services/widget_import_service.rb#L92)

Reply:
done, updated 👍 we need to create widget to get widget's id first and then we new to update (basically again) miq_schedule's condition(`MiqExpression`).